modules/container_runtime/data.tf

```diff
@@ -0,0 +1,28 @@
+data "aws_ami" "amazon-linux" {
```

Comment:
Instead of retrieving values via data resources, please use variables. Otherwise we'd need to define depends_on dependencies which causes other issues.